advisor-ui/src/components/OpportunityModal/OpportunityModal.css

```diff
@@ -0,0 +1,31 @@
+.oppmodal {
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.4);
+    color: #f5f5f5;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    left: 0;
+    overflow: auto;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1000;
+}
+
+.oppmodal .content {
+    align-items: flex-start;
+    background-color: white;
+    border: 1px solid black;
+    display: flex;
+    flex-direction: column;
+    gap: 3px;
+    height: 800px;
```

Comment:
It might be worth moving toward percents or some other "scaling" unit for sizes - that'll get you most of the way toward a responsive layout without having to do media queries or anything too verbose.